source/_integrations/obihai.markdown

```diff
@@ -15,16 +16,9 @@ ha_integration_type: integration
 
 The `obihai` integration allows you to view the call status for your [Obihai devices](https://www.obitalk.com/info/products#home_section).
 
-To enable `obihai` in your installation, add the following to your `configuration.yaml` file:
+{% include integrations/config_flow.md %}
 
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: obihai
-    host: 192.168.1.x
-```
-
-{% configuration %}
+{% configuration_basic %}
 host:
   description: IP Address of Obihai device
   required: true
```

Comment:
Configuration basic only has a `description` property on each of its items.
All others are not used or viable for use and should be removed.